src/components/Books.js

```diff
@@ -1,14 +1,30 @@
 import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import AddBook from './addBook';
+import { removeBook } from '../redux/books/books';
+
 /* eslint-disable react/prop-types */
 /* eslint-disable  react/destructuring-assignment */
 
```

Comment:
- [ ] When I started the application  I noticed a linters error came up and also in your code you disable a lot **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1:  

![Screenshot from 2022-11-10 23-27-22](https://user-images.githubusercontent.com/6615394/201227624-13bb4fe7-e82d-47bf-ad5e-a141b22d0599.png)